### ISSUE TITLE
Metatype 'int' not recognized in Compiler Specification after Ghidra version upgrade 

### ISSUE THREAD

**Issue by v-p-b** [NationalSecurityAgency/ghidra#6477]
**Describe the bug**

The following pentry specification:

```xml
<pentry minsize="1" maxsize="8" metatype="int">
```

Works in Ghidra 10, but results in an error in Ghidra 11.0.3.

The error is as follows:

```
Open failed: ghidra.program.model.lang.CompilerSpecNotFoundException: Exception reading [NAME]/default(file.cspec:123): Unknown type class: int
```

**To Reproduce**
1. Edit a cspec file to include the offending attribute in a pentry node
2. Open a file in Ghidra 11.0.3 using the modified cspec

**Expected behavior**

The cspec is parsed and the file is loaded for analysis.

**Screenshots**

N/A

**Attachments**

N/A

**Environment (please complete the following information):**
 - OS: Debian Linux 11
 - Java Version: 17
 - Ghidra Version: 11.0.3
 - Ghidra Origin: GitHub release

**Additional context**

I looked at the documentation of the latest release (I uploaded the generated docs [here](https://scrapco.de/ghidra_docs/cspec_parampass.html) ) and confirmed that the `metatype` attribute of `pentry` should accept `"int"` as a value:

> metatype 		(Optional) Restriction on datatype: unknown, float, int, uint, or ptr

**Comment by dev747368** [comment]
Yeah, doc is wrong (hasn't been updated since forever).  "int" type does not exist.

The code that parsed that value was looser, and used to ignore unknown values, but was changed 5/2023 in commit 191371675ae5906093f501461138e364f732487a.

**Comment by v-p-b** [comment]
This clarifies the behavior, thank you!

I suggest to keep this issue open until the documentation is updated (I'll try to open PR, but the linked commit is non-trivial so won't promise anything).
